common/config.yaml

```diff
@@ -27,9 +27,13 @@ detectors:
     DateFormatDetector:         1
 
 #Parallel Settings
-parallel:                       1
+parallel:                       0
 num_threads:                    5
 
+#Memory Logging Settings
+memlog_gen_features_frequency:  60
+memlog_classify_frequency:      1
```

Comment:
Just curious, but what's the rationale for having the classify frequency not be the same as the gen_features frequency?